readium/navigators/media/tts/src/main/java/org/readium/navigator/media/tts/TtsNavigator.kt

```diff
@@ -183,13 +183,13 @@ public class TtsNavigator<S : TtsEngine.Settings, P : TtsEngine.Preferences<P>,
         when (this) {
             TtsPlayer.State.Ready -> State.Ready
             TtsPlayer.State.Ended -> State.Ended
-            is TtsPlayer.State.Error -> this.toError()
+            is TtsPlayer.State.Failure -> this.toError()
         }
 
-    private fun TtsPlayer.State.Error.toError(): State.Error =
+    private fun TtsPlayer.State.Failure.toError(): State.Failure =
         when (this) {
-            is TtsPlayer.State.Error.ContentError -> State.Error(Error.ContentError(error))
-            is TtsPlayer.State.Error.EngineError<*> -> State.Error(Error.EngineError(error))
+            is TtsPlayer.State.Failure.ContentFailure -> State.Failure(Error.ContentError(error))
+            is TtsPlayer.State.Failure.EngineFailure<*> -> State.Failure(Error.EngineError(error))
```

Comment:
`Failure.Content` and `Failure.Engine` might be more consistent with the rest?

Reply:
Yes, good idea.